### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by abdday**
add new youtube playlist for Java

## What does this PR do?
Add resource

### Description
In this course you can learn tha java 17 programming language including all important cocepts like oop concepts , data structures and algoirthms
### Why is this valuable (or not)?
This course will handle the new features of java 17 and explain advanced topics like threads, sockts ect...
### How do we know it's really free?
its on youtube

### For book lists, is it a book? For course lists, is it a course? etc.
it is a course on youtbe for arabic language
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -178,7 +178,7 @@
 * [Learn JAVA Programming From Scratch In Arabic&rlm;](https://www.youtube.com/playlist?list=PLCInYL3l2AajYlZGzU_LVrHdoouf8W6ZN) - Adel Nasim
 * [Object-Oriented Programming JAVA in Arabic&rlm;](https://www.youtube.com/playlist?list=PLCInYL3l2AagY7fFlhCrjpLiIFybW3yQv) - Adel Nasim
 * [Programming 2 - Object Oriented Programming With Java](https://www.youtube.com/playlist?list=PL1DUmTEdeA6Icttz-O9C3RPRF8R8Px5vk) - محمد الدسوقى
-
+* [Java Tutorial for beginners- full course  -تعلم البرمجة- جافا](https://www.youtube.com/playlist?list=PLwAjM63H9bRuXIojpKDei4dVLRcvqP8V7) -  genial code
```
Comment (davorpa) [inline courses/free-courses-ar.md:181]: According to linter...
- Recover removed empty line at 181
- Alphabetize. Moving upwards the new resource
- Removed extra trailing whitespaces

```suggestion
* [Java Tutorial for beginners- full course -تعلم البرمجة- جافا](https://www.youtube.com/playlist?list=PLwAjM63H9bRuXIojpKDei4dVLRcvqP8V7) - genial code
* [Learn JAVA Programming From Scratch In Arabic&rlm;](https://www.youtube.com/playlist?list=PLCInYL3l2AajYlZGzU_LVrHdoouf8W6ZN) - Adel Nasim
* [Object-Oriented Programming JAVA in Arabic&rlm;](https://www.youtube.com/playlist?list=PLCInYL3l2AagY7fFlhCrjpLiIFybW3yQv) - Adel Nasim
* [Programming 2 - Object Oriented Programming With Java](https://www.youtube.com/playlist?list=PL1DUmTEdeA6Icttz-O9C3RPRF8R8Px5vk) - محمد الدسوقى

```

**Review by davorpa — CHANGES_REQUESTED** [review]
Needs solve linter faults. See suggestions and apply in concordance.

If you read CONTRIBUTING guidelines, the most common mistakes are alphabetization of listings or token whitespaces

**Comment by davorpa** [comment]
**TIP :writing_hand::** It's a good practice to fill Pull Request fields. It helps a bit we can understand your contribution better :hugs:. and then, increase maintainers productivity :rocket: 

> Why do you think it's important add it or ir valuable for our Arabic community?

File: courses/free-courses-ar.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-ar.md:181]: Not resolved yet

**Review by davorpa — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**